test/x3/Jamfile

```diff
@@ -8,107 +8,99 @@
 #   http://www.boost.org/LICENSE_1_0.txt)
 #==============================================================================
 
-# bring in rules for testing
-import testing ;
 import modules ;
 import feature ;
+import testing ;
 
 ###############################################################################
-project spirit_test
+
+project spirit-x3
     : requirements
         <include>.
+        <c++-template-depth>512
         <toolset>gcc:<cxxflags>-std=c++1y
-        <toolset>gcc:<cxxflags>-ftemplate-depth-512
         <toolset>clang:<cxxflags>-std=c++1y
-        <toolset>clang:<cxxflags>-ftemplate-depth-512
         <toolset>darwin:<cxxflags>-std=c++1y
-        <toolset>darwin:<cxxflags>-ftemplate-depth-512
-    :
-    :
     ;
```

Comment:
Why there is `-std=c++1y`? The c++ dialect is inject from outside into the test, isn't it?

Reply:
It is requirements, but actually they seems not to work as expected. I did not wrote it, but fixing this was not a plan for this PR.